src/engine/strat_engine/filesystem.rs

```diff
@@ -99,6 +99,13 @@ impl Filesystem for StratFilesystem {
             Err(e) => Err(EngineError::Engine(ErrorEnum::Error, e.description().into())),
         }
     }
+
+    fn devnode(&self) -> EngineResult<PathBuf> {
```

Comment:
We don't have a From<DmResult> currently, right?  Maybe I'm not understanding your comment.